gotests/main.go

```diff
@@ -41,6 +41,7 @@ var (
 	allFuncs      = flag.Bool("all", false, "generate tests for all functions and methods")
 	printInputs   = flag.Bool("i", false, "print test inputs in error messages")
 	writeOutput   = flag.Bool("w", false, "write output to (test) files instead of stdout")
+	templatePath  = flag.String("template", "", `path to a custom template set used to generate the tests`)
```

Comment:
Let's call this flag `template_dir` so that it's more clear that you need to point to a directory.

Reply:
👍